### PR TITLE
add documentation for magnifier

### LINKED ISSUE #1030: [FEATURE] add a map canvas magnifier
Original commit: https://github.com/qgis/QGIS/commit/1602c02df78a19688e2cf0df9aa5c6788581bbcd by 3nids

Merge pull request #3009 from pblottiere/magnifier

[FEATURE] add a map canvas magnifier

### PR DESCRIPTION
**Pull request by pblottiere**
This PR add documentation for the magnifier tool.

Note that this feature is for 2.16.

Fix #1030

### CODE REVIEW

**Comment by yjacolin** [comment]
Could you add some index:: tag ? LGTM

File: source/docs/user_manual/introduction/qgis_gui.rst [hunk 1]
```diff
@@ -766,9 +766,14 @@ It shows the scale of the map view. If you zoom in or out, QGIS shows you the
 current scale. There is a scale selector, which allows you to choose among
 :ref:`predefined and custom scales <predefinedscales>` to assign to the map view.
 
-To the right of the scale display you can define a current clockwise rotation for
-your map view in degrees.
+On the right side of the scale display you can define a current clockwise
+:index:`magnification` level for your map view. The level is expressed as a percentage.
+If the :guilabel:`Magnifier` has a level of 100%, then the current map is not
+magnified. A default value for this level is available within
+:menuselection:`Settings --> Options --> Rendering --> Rendering behaviour`.
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/qgis_gui.rst]: behavior (to be en-US compliant and as  in the gui)

**Comment by DelazJ** [comment]
Having read the discussion about the PR of this tool and reading the doc now, I'm still unable to understand its objective. Sorry.
Would that be possible to explain what does/is the magnifier? A short description of its rationale (vs zoom in/zoom out for example) can be of great value. I think that user should not have to use the tool before understanding what it's about. Description should be sufficient or help enough.

**Comment by SrNetoChan** [comment]
I agree with @Delaz. Tho I know what the tool does, it's not clear in the documentation,

**Comment by DelazJ** [comment]
@SrNetoChan you forgot the "J" of DelazJ. :) Would be nice if you can explain me how that works later.

File: source/docs/user_manual/introduction/qgis_gui.rst [hunk 2]
```diff
@@ -766,9 +766,14 @@ It shows the scale of the map view. If you zoom in or out, QGIS shows you the
 current scale. There is a scale selector, which allows you to choose among
 :ref:`predefined and custom scales <predefinedscales>` to assign to the map view.
 
-To the right of the scale display you can define a current clockwise rotation for
-your map view in degrees.
+On the right side of the scale display you can define a current clockwise
```
Comment (yjacolin) [inline source/docs/user_manual/introduction/qgis_gui.rst]: `clockwise` need to be removed

File: source/docs/user_manual/introduction/qgis_gui.rst [hunk 3]
```diff
@@ -766,9 +766,14 @@ It shows the scale of the map view. If you zoom in or out, QGIS shows you the
...
+:index:`magnification` level for your map view. The level is expressed as a percentage.
+If the :guilabel:`Magnifier` has a level of 100%, then the current map is not
```
Comment (yjacolin) [inline source/docs/user_manual/introduction/qgis_gui.rst]: Better to explain why users want to setup another value than 100% like 150% or 50%.

Does it interesting only when users setup style with scale rendering?

**Comment by pblottiere** [comment]
I tried to write a clearer explanation for the magnifier in a few words.

Below a GIF that I've used in the 2.16 changelog to explain the magnifier behaviour :

4991ffccebf8da94cd01acc1139f52d389731549

Let me know if I'm still unclear :)

And very sorry for the delay...

**Comment by DelazJ** [comment]
@pblottiere Thanks for your explanation. @yjacolin and @SrNetoChan had explained it to me last day and your text looks really clearer to me.
Not a problem for the delay

**Pull request merged**